diligent-academy-assessments/backend/database-challenge/README.md

```diff
@@ -0,0 +1,22 @@
+# Database challenge
+
+## Principles and rules
+
+- Please use the prepared `.sql` files to solve the tasks.
+
+## Tasks
+
+1. Create the student table with the following fields: `name` and `email`. What columns would you create to represent
+   them? Which types and constraints would make sense for a user table?
```

Comment:
Ooops, I made a mistake here. It should refer to the `students` table not the `user` table:
```suggestion
1. Create the student table with the following fields: `name` and `email`. What columns would you create to represent
   them? Which types and constraints would make sense for this table?
```